### ISSUE TITLE
Dynamic Link Library Error in 1.0.5

### ISSUE THREAD

**Issue by tessaleejohnson** [RcppCore/Rcpp#1103]
After updating Rcpp from 1.0.4 to 1.0.5, I receive the following error when updating the package documentation for any local packages on my machine (the error disappears if I revert to Rcpp version 1.0.4):

```
devtools::document("C:/Users/tjohnson/NewPackage")
Updating NewPackage documentation
Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Users/tjohnson/Documents/R/win-library/4.0/Rcpp/libs/x64/Rcpp.dll':
  LoadLibrary failure:  The specified procedure could not be found.
```

In addition, I receive the following popup:
![dll_error](https://user-images.githubusercontent.com/59889043/87160369-db630400-c290-11ea-88c4-2484967592d7.png)

(Image text:
100% downloaded: rsession.exe - Entry Point Not Found

The procedure entry point EXTPTR_PTR could not be located in the dynamic link library C:\Users\tjohnson\Documents\R\win-library\4.0\Rcpp\libs\x64\Rcpp.dll.)

Session Info:

```
- Session info ---------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 4.0.0 (2020-04-24)
 os       Windows 10 x64              
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United States.1252  
 ctype    English_United States.1252  
 tz       America/New_York            
 date     2020-07-10                  

- Packages -------------------------------------------------------------------------------------------------------
 package     * version date       lib source        
 assertthat    0.2.1   2019-03-21 [1] CRAN (R 4.0.0)
 backports     1.1.8   2020-06-17 [1] CRAN (R 4.0.0)
 callr         3.4.3   2020-03-28 [1] CRAN (R 4.0.0)
 cli           2.0.2   2020-02-28 [1] CRAN (R 4.0.0)
 crayon        1.3.4   2017-09-16 [1] CRAN (R 4.0.0)
 desc          1.2.0   2018-05-01 [1] CRAN (R 4.0.0)
 devtools      2.3.0   2020-04-10 [1] CRAN (R 4.0.2)
 digest        0.6.25  2020-02-23 [1] CRAN (R 4.0.0)
 ellipsis      0.3.1   2020-05-15 [1] CRAN (R 4.0.2)
 evaluate      0.14    2019-05-28 [1] CRAN (R 4.0.0)
 fansi         0.4.1   2020-01-08 [1] CRAN (R 4.0.0)
 fs            1.4.2   2020-06-30 [1] CRAN (R 4.0.2)
 glue          1.4.1   2020-05-13 [1] CRAN (R 4.0.2)
 htmltools     0.5.0   2020-06-16 [1] CRAN (R 4.0.2)
 knitr         1.29    2020-06-23 [1] CRAN (R 4.0.2)
 magrittr      1.5     2014-11-22 [1] CRAN (R 4.0.0)
 memoise       1.1.0   2017-04-21 [1] CRAN (R 4.0.0)
 pkgbuild      1.0.8   2020-05-07 [1] CRAN (R 4.0.2)
 pkgload       1.1.0   2020-05-29 [1] CRAN (R 4.0.2)
 prettyunits   1.1.1   2020-01-24 [1] CRAN (R 4.0.0)
 processx      3.4.3   2020-07-05 [1] CRAN (R 4.0.2)
 ps            1.3.3   2020-05-08 [1] CRAN (R 4.0.2)
 R6            2.4.1   2019-11-12 [1] CRAN (R 4.0.0)
 remotes       2.1.1   2020-02-15 [1] CRAN (R 4.0.0)
 rlang         0.4.7   2020-07-09 [1] CRAN (R 4.0.0)
 rmarkdown     2.3     2020-06-18 [1] CRAN (R 4.0.2)
 rprojroot     1.3-2   2018-01-03 [1] CRAN (R 4.0.0)
 rstudioapi    0.11    2020-02-07 [1] CRAN (R 4.0.0)
 sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 4.0.0)
 stringi       1.4.6   2020-02-17 [1] CRAN (R 4.0.0)
 stringr       1.4.0   2019-02-10 [1] CRAN (R 4.0.0)
 testthat      2.3.2   2020-03-02 [1] CRAN (R 4.0.0)
 usethis       1.6.1   2020-04-29 [1] CRAN (R 4.0.2)
 withr         2.2.0   2020-04-20 [1] CRAN (R 4.0.0)
 xfun          0.15    2020-06-21 [1] CRAN (R 4.0.2)
 yaml          2.2.1   2020-02-01 [1] CRAN (R 4.0.0)

[1] C:/Users/tjohnson/Documents/R/win-library/4.0
[2] C:/Program Files/R/R-4.0.0/library
```

**Comment by eddelbuettel** [comment]
It is related to an inadvertent change in R 4.0.0 that is fixed in R 4.0.2.  Simply upgrade to R 4.0.2 --- for which you do NOT need to reinstall any library.  It is not really an issue with Rcpp (though we since added one patch to be even more defensive about this).

Sorry about the woes, but we too only became aware of this very recently and not in time for the R 1.0.5 release.

**Comment by tessaleejohnson** [comment]
Perfect, thanks!
